### PR TITLE
Release 2.4.0

### PR DESCRIPTION
**Pull request by thiemowmde**

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
This updated requirement kinda sucks. Is it really needed? If the only compat issue here is in the tests of this component, then you can resolve it there, without effecting users of this component that do not care about the hash.

**Comment by thiemowmde** [comment]
I also was not happy with this. I had an other look and believe I found a really nice solution. What do you think?

File: tests/unit/Serializers/SnakSerializerTest.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
 class SnakSerializerTest extends DispatchableSerializerTest {
 
 	protected function buildSerializer() {
-		return new SnakSerializer( new DataValueSerializer() );
+		return new SnakSerializer( new DataValueSerializer(), false );
```
Comment (bekh6ex) [inline tests/unit/Serializers/SnakSerializerTest.php]: Could you name this `false`? Otherwise it quiet hard to read

Comment (thiemowmde) [inline tests/unit/Serializers/SnakSerializerTest.php]: Done.

**Pull request merged**